time data
---
mask_rcnn/SemanticObject[] semantic_objects
bool found